app/models/edition.rb

```diff
@@ -130,6 +131,10 @@ def editable?
     !live? && !scheduled?
   end
 
+  def first?
```

Comment:
This looks useful, and I can see you've replaced it in a few places already. Do you think we could have a subsequent PR to make use of it throughout the rest app? Possible use case: https://github.com/alphagov/content-publisher/blob/master/app/views/documents/show/_contents.html.erb#L10

Reply:
Oh sure, I'd looked for instances of `edition.number == 1` but hadn't considered where the `<` and `>` operators may be used.

Reply:
I've searched for the other instances and changed them - hopefully I found them all 🤞